### ISSUE TITLE
[iOS] "ons-lazy-repeat" does not work in "ons-template".

### ISSUE THREAD

**Issue by keijiodagawa** [OnsenUI/OnsenUI#1914]
__Environment__ 

```
[Core]
    Onsen UI 2.2.1

[Framework]
    angular1 1.5.8

[Framework binding]
    angular-onsenui 2.2.1

[Additional libraries]
    none

[Platform]
    Mobile - iOS 9.2

```

__Encountered problem__
[iOS] "ons-lazy-repeat" does not work in "ons-template".

__How to reproduce__
https://codepen.io/keiji/pen/NpVWNZ

**Comment by asial-matagawa** [comment]
@keijiodagawa 
Thank you for reporting.
This issue might be fixed by #1892.
Please be patient until Onsen UI 2.2.2 (or later) is released.

**Comment by asial-matagawa** [comment]
@keijiodagawa 
I made sure that this issue has been fixed by #1892.
You can fix this problem by updating `onsenui` to 2.2.3 or later.

Closing this.
